regression/test.pl

```diff
@@ -71,7 +71,9 @@ ($$$$$)
 
   $options =~ s/$ign//g if(defined($ign));
 
-  my $output = $input;
+  my $descriptor = $test;
+  $descriptor =~ s/^.*\///;
```

Comment:
@smowton I'm in minority here, and so I've already force-pushed the File::Basename version, which is of course clearer. What I was basing of comments on is that it's not the first instance of that of that kind of expression in the perl files we have so I assumed it a norm.